fulp_modules/bloodsuckers/code/powers/masquerade.dm

```diff
@@ -53,8 +53,8 @@
 	// Falsifies Genetic Analyzers
 	REMOVE_TRAIT(user, TRAIT_GENELESS, SPECIES_TRAIT)
 	var/obj/item/organ/heart/vampheart/H = user.getorganslot(ORGAN_SLOT_HEART)
-	var/obj/item/organ/eyes/bloodsucker/E = user.getorganslot(ORGAN_SLOT_EYES)
-	E.flash_protect = 0
+	var/obj/item/organ/eyes/E = user.getorganslot(ORGAN_SLOT_EYES)
+	E.flash_protect += 1
```

Comment:
you can activate this power, get your eyes removed, deactivate it, and get your eyes reinserted into yourself to improve the flash_protect of your eyes by 1. you can repeat that process until your eyes become as flash-resistant as you feel that they should be.

Reply:
Thats better than getting +1 free flash protection from activating/deactivating as a moth